aiohttp/connector.py

```diff
@@ -701,6 +701,8 @@ def next_addrs(self, key: Tuple[str, int]) -> List[Dict[str, Any]]:
     def expired(self, key: Tuple[str, int]) -> bool:
         if self._ttl is None:
             return False
+        elif self._ttl == 0:
+            return True
```

Comment:
I agree. Change to `is not None` per suggestion to fix crashing on `ttl = 0`.

Also, do you think it makes sense in case when user passes number <=0 to `ttl_dns_cache` to override `use_dns_cache` to `False` as behaviour will be the same but performance would be better?

Reply:
I think if we were designing it from scratch it would make sense to have only 1 option, rather than both. Maybe it's worth removing the `use_dns_cache` option in v4...